config.go

```diff
@@ -47,24 +47,24 @@ func parseJSON(data json.RawMessage) (Config, error) {
 // GetConsolidatedConfig combines {default config values + JSON config +
 // environment vars config values}, and returns the final result.
 func GetConsolidatedConfig(
-	jsonRawConf json.RawMessage,
+	jsonRawConf json.RawMessage, env map[string]string, url string,
```

Comment:
```suggestion
	jsonRawConf json.RawMessage, env map[string]string,
```

I'd suggest removing the URL parameter from this line as it's not used. It looks like this came from https://github.com/grafana/xk6-output-influxdb/blob/main/pkg/influxdb/config.go#L116 where the influxdb location can be specified as a URI. I don't believe there's the same concept for Timestream, and to keep this PR focussed on just the environment parsing, I'd leave this out.

Reply:
Ah, that's right. Let me remove it.